pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -162,18 +163,21 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 			attr.deserved.Add(remaining.Clone().Multi(float64(attr.weight) / float64(totalWeight)))
 
 			if attr.capability != nil && !attr.deserved.LessEqualStrict(attr.capability) {
-				attr.deserved = helpers.Min(attr.deserved, attr.capability)
-				attr.deserved = helpers.Min(attr.deserved, attr.request)
+				attr.deserved = attr.capability
+				//attr.deserved = helpers.Min(attr.deserved, attr.capability)
+				//attr.deserved = helpers.Min(attr.deserved, attr.request)
 				meet[attr.queueID] = struct{}{}
 				klog.V(4).Infof("queue <%s> is meet cause of the capability", attr.name)
-			} else if attr.request.LessEqualStrict(attr.deserved) {
-				attr.deserved = helpers.Min(attr.deserved, attr.request)
-				meet[attr.queueID] = struct{}{}
-				klog.V(4).Infof("queue <%s> is meet", attr.name)
-			} else {
-				attr.deserved.MinDimensionResource(attr.request)
-				klog.V(4).Infof("Format queue <%s> deserved resource to <%v>", attr.name, attr.deserved)
 			}
+			//else if attr.request.LessEqualStrict(attr.deserved) {
```

Comment:
does these changes related with the issue, can you explain more?

Reply:
This logic is a bit confuse, but it is not related to this issue. I have keep it as the original.

Reply:
Actually I also want to know why the judgment `attr.request.LessEqualStrict(attr.deserved)` is needed.

Reply:
That is what i want to know too 😂 ,  i think we can fix it in the next PR.

Reply:
l think so. Any thoughts why this judgment is needed? /cc @Thor-wl @k82cn

Reply:
/cc @Thor-wl

Reply:
> /cc @Thor-wl

I'm sorry for not have a better understanding.